### PR TITLE
misc: bypassing nonzero return in Dockerfile

### PR DESCRIPTION
**Pull request by haileynam**
issue #1766 related added bypassing command to make install properly

### CODE REVIEW

**Review by namhyung — COMMENTED** [review]
Too many sign-off from the same person. :)

Please describe the issue in the commit message directly instead of referring the github issue.  The issue link is just to close it or add a reference for github and we want to keep the change log in the git history.

File: misc/docker/ubuntu/16.04/Dockerfile [hunk 1]
```diff
@@ -9,7 +9,7 @@ RUN if [ "$test" = "yes" ] ; then \
         && ./misc/install-deps.sh -y \
         && ./configure && make ASAN=1 && make ASAN=1 unittest; \
     else \
-        cd /usr/src/uftrace && ./misc/install-deps.sh -y && ./configure && make && make install; \
+        cd /usr/src/uftrace && ./misc/install-deps.sh -y && ./configure && make && make install || : && make install; \
```
Comment (namhyung) [inline misc/docker/ubuntu/16.04/Dockerfile:12]: I think we should fix the document not to fail.

File: misc/docker/ubuntu/18.04/Dockerfile [hunk 2]
```diff
@@ -9,7 +9,7 @@ RUN if [ "$test" = "yes" ] ; then \
         && ./misc/install-deps.sh -y \
         && ./configure && make ASAN=1 && make ASAN=1 unittest; \
     else \
-        cd /usr/src/uftrace && ./misc/install-deps.sh -y && ./configure && make && make install; \
+        cd /usr/src/uftrace && ./misc/install-deps.sh -y && ./configure && make && make install || : make install; \
```
Comment (namhyung) [inline misc/docker/ubuntu/18.04/Dockerfile:12]: Ditto.

**Review by namhyung — COMMENTED** [review]
Please break long lines around 72.

**Comment by haileynam** [comment]
I will close this PR since it is fixed by changing the pandoc syntax problem.

**Pull request closed without merging**